nxc/protocols/smb/kerberos.py

```diff
@@ -72,96 +67,80 @@ def kerberos_login_with_S4U(domain, hostname, username, password, nthash, lmhash
     # key (Section 5.5.1)
     encryptedEncodedAuthenticator = cipher.encrypt(sessionKey, 7, encodedAuthenticator, None)
 
-    apReq['authenticator'] = noValue
-    apReq['authenticator']['etype'] = cipher.enctype
-    apReq['authenticator']['cipher'] = encryptedEncodedAuthenticator
+    apReq["authenticator"] = noValue
+    apReq["authenticator"]["etype"] = cipher.enctype
+    apReq["authenticator"]["cipher"] = encryptedEncodedAuthenticator
 
     encodedApReq = encoder.encode(apReq)
 
     tgsReq = TGS_REQ()
 
-    tgsReq['pvno'] = 5
-    tgsReq['msg-type'] = int(constants.ApplicationTagNumbers.TGS_REQ.value)
+    tgsReq["pvno"] = 5
+    tgsReq["msg-type"] = int(constants.ApplicationTagNumbers.TGS_REQ.value)
 
-    tgsReq['padata'] = noValue
-    tgsReq['padata'][0] = noValue
-    tgsReq['padata'][0]['padata-type'] = int(constants.PreAuthenticationDataTypes.PA_TGS_REQ.value)
-    tgsReq['padata'][0]['padata-value'] = encodedApReq
+    tgsReq["padata"] = noValue
+    tgsReq["padata"][0] = noValue
+    tgsReq["padata"][0]["padata-type"] = int(constants.PreAuthenticationDataTypes.PA_TGS_REQ.value)
+    tgsReq["padata"][0]["padata-value"] = encodedApReq
 
     # In the S4U2self KRB_TGS_REQ/KRB_TGS_REP protocol extension, a service
     # requests a service ticket to itself on behalf of a user. The user is
-    # identified to the KDC by the user's name and realm.
+    # identified to the KDC by the user"s name and realm.
```

Comment:
Not necessary for merge, but it looks like the replace all replaced on that shouldn't be lol.

In the future the Ruff linter will properly fix these (with --fix) or let people know.